sonar-application/src/main/assembly/conf/sonar.properties

```diff
@@ -342,6 +342,11 @@
 #sonar.path.data=data
 #sonar.path.temp=temp
 
+# Telemetry - Share anonymous SonarQube statistics
+# By sharing anonymous SonarQube statistics, you help us understand how SonarQube is used so we can improve the plugin to work even better for you.
+# We don't collect source code or IP addresses. And we don't share the data with anyone else.
```

Comment:
Is there a documentation listing what is sent ? If yes, then providing a link to the document would make sense here.

Reply:
no, there's no documentation. I'll point to WS `api/system/info`, the `Statistics` field